README.md

```diff
@@ -1 +1,65 @@
-# volttron-lib-tagging
\ No newline at end of file
+[![Eclipse VOLTTRON™](https://img.shields.io/badge/Eclips%20VOLTTRON--red.svg)](https://eclipse-volttron.readthedocs.io/en/latest/)
+![Python 3.10](https://img.shields.io/badge/python-3.10-blue.svg)
+![Python 3.11](https://img.shields.io/badge/python-3.11-blue.svg)
+[![Run Pytests](https://github.com/eclipse-volttron/volttron-lib-tagging/actions/workflows/run-tests.yml/badge.svg)](https://github.com/eclipse-volttron/volttron-lib-tagging/actions/workflows/run-tests.yml)
+[![pypi version](https://img.shields.io/pypi/v/volttron-lib-tagging.svg)](https://pypi.org/project/volttron-lib-tagging/)
+
+VOLTTRON base tagging library that provide common tagging api for users to associate haystack based tags and values to 
```

Comment:
do we want to specifically call out haystack?  provide a link to them?

Reply:
looks like a link is below. Clean up because this isn't an agent.

Reply:
I am not sure what you mean. I cleaned up overall. Let me know if this works